### DISCUSSION TITLE
How can I specifies attachment filename of test step in cucumber-js

### DISCUSSION THREAD

**Discussion by Jungwooyung** [allure-framework/allure2#2555] [Questions & Support] [answered]
Hi, everyone. I need your help. 
I want to specifies the attachment filename like "input message", not just like same "attachement" when I attach the input/output json file of step. 

### ↓ This is my current allure test step report image - just "attachment". 
![image](https://github.com/allure-framework/allure2/assets/152781664/7099e5a8-7b0d-45e4-b511-3061752a2d78)

### ↓ This is my javascript code 
```javascript
if (messageReportable) {
	world.attach(JSON.stringify(inputData, null, 4), "application/json");
	world.attach(JSON.stringify(outputJson, null, 4), "application/json");
}
```

**Comment by baev** [comment, marked as answer]
the issue https://github.com/allure-framework/allure-js/issues/801

already fixed in https://github.com/allure-framework/allure-js/pull/909

**Reply by Jungwooyung** [reply]
Your help is greatly appreciated.
